scripts/teamDataScript.js

```diff
@@ -135,74 +136,75 @@ function loadPit()
     if (currTeam == 0)
     {
-        $('#pitQuestionTable').html('');
+        $('#pitQuestionTableHeader').html('');
+        $('#pitQuestionTableBody').html('');
         $('#pitTableConfirmation').text('No team selected.');
         return;
     }
 
-    var tableHeader = '<tr> <th><b>Question</b></th> <th><b>Answer</b></th> </tr>';
-    $('#pitQuestionTable').html(tableHeader); //default start of table
+    var tableHeader = '<div class="w3-col m6 s12 w3-center"> <h4>Question</h4> </div>'+
+                        '<div class="w3-col m6 s12 w3-center"> <h4>Answer</h4> </div>';
+    $('#pitQuestionTableHeader').html(tableHeader); //default start of table
 
+    $('#pitQuestionTableBody').html('');
     var localStorageObjName = pitAnswerObjName(season,event,currTeam);
     var team_QA_pairs = JSON.parse(localStorage.getItem(localStorageObjName)); //get QA pairs that may/may not have already been stored
-    var teamsAlreadyStoredBool = false;
-    if (checkForNull(team_QA_pairs))
+    
+    var alreadyStored = false;
+    var QA_keys = [];
+    if (checkForNull(team_QA_pairs)) //if data is stored
     {
-        teamsAlreadyStoredBool = true;
+        alreadyStored = true;
+        QA_keys = Object.keys(team_QA_pairs);
     }
 
-    for (var i = 0; i < questions.length;i++)
+    for (var i = 0; i < questions.length; i++) //iterate through questions array in cookie
     {
         var question = questions[i];
-        if (question.trim().length == 0) return;
-        
-        //loop through already stored local object: if question has an answer already stored then put that in the input box. Else, leave it empty (to fill in)
-        var potentialAnswer = '';
-        if (teamsAlreadyStoredBool == false) //len of loc storage obj == 0, nothing stored yet
+        var answer = ''; //default, in case something is stored
+        if (alreadyStored)
         {
-            potentialAnswer = '';
-        } else //len of loc storage obj = 1, something stored inside, now find answer to curr question
-        {
-            for (var j = 0; j < team_QA_pairs.length; j++)
+            if (QA_keys.includes(question)) //if answer is saved for it yet
             {
-                var pair = team_QA_pairs[j];
-                var answer = pair[question];
-                if (!checkForNull(answer))
-                {
-                    continue;
-                }
-                potentialAnswer = answer;
+                answer = team_QA_pairs[question];
             }
         }
 
-        var inputHTML = `<input type="text" value="${potentialAnswer}" class="answer">`; //pot answer is either '' or what is stored
-        var newTRHtml = `<tr><td class='question scoutingTableRow'>${question}</td><td class='scoutingTableRow'>${inputHTML}</td></tr>`;
+        var classText = 'w3-col l6 w6 s12 question w3-center'; 
+        var inputHTML = `<textarea width='50em' style='resize:vertical' value='${answer}' class='answer'>${answer}</textarea>`; //textarea
```

Comment:
use `width:80%` (or some high percentage) in the style string this will make the box dynamically almost as big as the area it can fit in (width wise).